app/models/movie_cast.rb

```diff
@@ -1,36 +1,29 @@
+# frozen_string_literal: true
```

Comment:
This file is a good example of the changes I made to the other files in this PR. 
* move the `attr_accessor` to the top
* switch to keyword args
* use `map` to return a new array instead of shoveling into an array
* build the `new` object in the `map` instead of setting up all of the attributes first
* remove instance variables from the `map` block